src/requirements/decorated-requirements.json

```diff
@@ -217243,97 +217283,12 @@
     }
   },
   "minor": {
-    "APPLIEDECON": {
-      "name": "Applied Economics",
-      "schools": [
-        "AG",
-        "BU"
-      ],
-      "requirements": [
-        {
-          "name": "Introductory Economics",
-          "description": "ECON 1110 and ECON 1120",
-          "source": "https://business.cornell.edu/programs/undergraduate/minors/applied-economics/",
-          "fulfilledBy": "courses",
-          "perSlotMinCount": [
-            1,
-            1
-          ],
-          "slotNames": [
-            "ECON 1110",
-            "ECON 1120"
-          ],
-          "courses": [
-            [
-              104,
-              105,
-              203,
-              350025
-            ],
-            [
-              106,
-              203,
-              350038
-            ]
-          ]
-        },
-        {
-          "name": "Economic Theory or Practice",
-          "description": "Choose one of the following: AEM 2600, ECON 3030, or PAM 2000",
-          "source": "https://business.cornell.edu/programs/undergraduate/minors/applied-economics/",
-          "fulfilledBy": "courses",
-          "perSlotMinCount": [
-            1
-          ],
-          "slotNames": [
-            "Course"
-          ],
-          "courses": [
-            [
-              350335,
-              351447,
-              364480
-            ]
-          ]
-        },
-        {
-          "name": "Applied Economics",
-          "description": "Three of the following courses are required. Please note that 3 credits must be Johnson College of Business (JCB) courses, which include any course code of AEM, HADM, NBA, or NCC.",
-          "source": "https://business.cornell.edu/programs/undergraduate/minors/applied-economics/",
-          "fulfilledBy": "courses",
-          "perSlotMinCount": [
-            3
-          ],
-          "slotNames": [
-            "Course"
-          ],
-          "courses": [
-            [
-              351930,
-              351937,
-              351969,
-              352024,
-              352040,
-              352070,
-              354511,
-              359935,
-              361193,
-              363567,
-              363568,
-              365803,
-              367765,
-              370423,
-              371372
-            ]
-          ]
-        }
-      ]
-    },
     "AEROSPACE": {
       "name": "Aerospace Engineering",
       "schools": [
         "EN"
       ],
+      "abbrev": "Aero Eng",
```

Comment:
can we exclude the abbrev from the computed json like we do with advisors?

Reply:
it was included in req-gen which i had to run to pass the requirements check

Reply:
yeah but we can exclude it from req-gen